nailgun/entities.py

```diff
@@ -6789,33 +6789,31 @@ def __init__(self, server_config=None, **kwargs):
             'to': entity_fields.IPAddressField(),
             'vlanid': entity_fields.StringField(),
             'mtu': entity_fields.IntegerField(min_val=68, max_val=4294967295),
+            'boot_mode': entity_fields.StringField(
```

Comment:
Bonus karma if you sort all fields alphabetically 🙂

Reply:
done, pls. resolve the conversation if happy with the changes